Source/WebCore/style/Styleable.h

```diff
@@ -204,20 +200,34 @@ class WeakStyleable {
 
     explicit operator bool() const { return !!m_element; }
 
+    bool operator==(const WeakStyleable& other) const
+    {
+        return (element() == other.element() && pseudoElementIdentifier() == other.pseudoElementIdentifier());
+    }
+
     WeakStyleable& operator=(const Styleable& styleable)
     {
         m_element = styleable.element;
         m_pseudoElementIdentifier = styleable.pseudoElementIdentifier;
         return *this;
     }
 
+    WeakStyleable(const Styleable& styleable)
+    {
+        m_element = styleable.element;
+        m_pseudoElementIdentifier = styleable.pseudoElementIdentifier;
+    }
```

Comment:
I'm curious what in this patch uses this.

Reply:
No longer necessary.

Reply:
Can we remove it if that's the case?

Reply:
Apologies this is necessary for the `setSource(Styleable::fromElement(*source))` call.